content/docs/reference/crosswalk/aws/ecs.md

```diff
@@ -298,25 +296,26 @@ https://docs.aws.amazon.com/AmazonECS/latest/developerguide/service_definition_p
 
 ## Building and Publishing Docker Images Automatically
 
-Containers with Pulumi Crosswalk for AWS ECS are far more flexible than just accepting a pre-existing image URL,
-however, and can even refer to a `Dockerfile` on disk so that you don't need to arrange to build and publish
+Containers with Pulumi Crosswalk for AWS ECS are far more flexible than just accepting a preexisting image URL,
+and can even refer to a `Dockerfile` on disk so you do not need to build and publish
 it separately ahead of time. This makes it very easy to use private registrations for your ECS workloads.
 
-For example, `fromPath` will run a Docker build in that path, push the result up to an ECR repository, and then pass
+For example, [`fromPath`](/docs/reference/pkg/nodejs/pulumi/awsx/ecs/#Image-fromPath") will run a `docker build` in that path, push the result up to the ECR repository that specified in the first argument, and then pass
 the private ECR repostory path to the container:
 
 ```typescript
 const task = new awsx.ecs.FargateTaskDefinition("task", {
     containers: {
         nginx: {
-            image: awsx.ecs.Image.fromPath("<path-to-dockerfile>"),
+            image: awsx.ecs.Image.fromPath("<ecr-repository-name>", "/path/to/Dockerfile/"
+),
```

Comment:
Nit, line break here felt odd (I'd put the `),` on the same line as `image: ...`).